ios_ui/MovieRec/Recommendations.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  Recommendations.swift
+//  MovieRec
+//
+//  Created by Yujin Cho on 8/19/17.
+//  Copyright © 2017 Yujin Cho. All rights reserved.
+//
+
+import os.log
+import UIKit
+import SwiftyJSON
+
+protocol RecTableDelegate: class {
+    func endLoadingOverlay()
+    func refreshTable()
+}
+
+class Recommendations {
+    var recommendations = [Recommendation]()
+    var timer: DispatchSourceTimer?
+    weak var delegate: RecTableDelegate?
+    
+    enum TrainingStatus: String {
```

Comment:
I typically don't use enums when theres only one case, is this going to expand in the future?